<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="23.8s"
		durationms="23803"
		type="root"
		rid="1d5891d7b35deff4"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="Z440"
		rxversion="8.0.0.0"
		host="Z440-PC"
		osversion="Windows 7 Service Pack 1 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="en-US"
		dtnlanguage="ko-KR"
		screenresolution="1920x1080"
		timestamp="2021-02-18 오후 5:21:07"
		timestampiso="2021-02-18T08:21:07"
		endtime="2021-02-18 오후 5:21:31"
		timeoutfactor="1"
		module="NormalJob">
		<detail>
			NormalJob
		</detail>
		<varbindings>
		</varbindings>
		<item
			timeRelativeToTestSuiteStartTime="00:23.803"
			timeRelativeToTestModuleStartTime="00:23.804"
			timeWallClock="17:21:31"
			time="00:23.803"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
